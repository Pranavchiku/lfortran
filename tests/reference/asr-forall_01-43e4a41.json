{
    "basename": "asr-forall_01-43e4a41",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/forall_01.f90",
    "infile_hash": "a482aaee28abbbe45149c269745483f2ede65c21ea57212e6a647019",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-forall_01-43e4a41.stdout",
    "stdout_hash": "fb127512b5bbbb7aec1e9b5241882007f49a5beda2b97f4564941c17",
    "stderr": "asr-forall_01-43e4a41.stderr",
    "stderr_hash": "93f6a16574c23e1bd887948b67e394930c50a89bbd125945937d6053",
    "returncode": 0
}