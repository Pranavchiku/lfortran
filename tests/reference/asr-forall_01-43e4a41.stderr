warning: Implied save feature
 --> tests/../integration_tests/forall_01.f90:3:1
  |
3 | integer :: ivec(3), i
  | ^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/../integration_tests/forall_01.f90:3:1
  |
3 | integer :: ivec(3), i
  | ^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
